### PR TITLE
Fixed a typo in name menu

### PR DESCRIPTION
**Pull request by ABODFTW**
Fixed a typo in the name entity menu to be more descriptive

### CODE REVIEW

File: src/entity/name.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export class Name implements FakerEntity {
     return [
       'firstName',
       'lastName',
-      'findName',
+      'fullName',
```
Comment (deerawan) [inline src/entity/name.ts:12]: I agree that `fullName` is better name but unfortunately `findName` is not just an arbitrary name, it is a method name in Faker. 

image

Comment (ABODFTW) [reply src/entity/name.ts:12]: I see, that makes sense, shall I change it to what you suggested, or just close the pr?

Comment (deerawan) [reply src/entity/name.ts:12]: if you want to make the changes, I'm welcoming that :D but it means that the changes will not only be applied for `findName` but also the other parts.

**Comment by deerawan** [comment]
@ABODFTW I closed the PR. Feel free to re-open if you got a better approach. Thank you 🥰

**Pull request closed without merging**